tests/tests/tier0/bluechi-propogating-service-failure/test_bluechi_propogating_service_failure.py

```diff
@@ -0,0 +1,55 @@
+# SPDX-License-Identifier: LGPL-2.1-or-later
+
+import logging
+
+from typing import Dict
+from bluechi_test.test import BluechiTest
+from bluechi_test.machine import BluechiControllerMachine, BluechiAgentMachine
+from bluechi_test.config import BluechiControllerConfig, BluechiAgentConfig
+from bluechi_test.service import Option, Section, SimpleRemainingService
+
+LOGGER = logging.getLogger(__name__)
+
+NODE_FOO = "node-foo"
+NODE_BAR = "node-bar"
+
+
+def exec(ctrl: BluechiControllerMachine, nodes: Dict[str, BluechiAgentMachine]):
+
+    node_foo = nodes[NODE_FOO]
+    node_bar = nodes[NODE_BAR]
+
+    service = SimpleRemainingService("target.service")
+    service.set_option(Section.Service, Option.ExecStart, "/bin/false")
+
+    node_bar.install_systemd_service(service)
+
+    service = SimpleRemainingService("requesting.service")
+    service.set_option(Section.Unit, Option.BindTo, "bluechi-proxy@node-bar_target.service")
+    service.set_option(Section.Unit, Option.After, "bluechi-proxy@node-bar_target.service")
+    service.set_option(Section.Service, Option.ExecStart, "/bin/false")
```

Comment:
```suggestion
    service.set_option(Section.Service, Option.ExecStart, "/bin/true")
```
This should succeed usually to demonstrate that it really fails due to the `BindsTo=` dependency on the failing target.service.